src/ra_server.erl

```diff
@@ -195,6 +197,7 @@
                               max_pipeline_count => non_neg_integer(),
                               ra_event_formatter => {module(), atom(), [term()]},
                               counter => counters:counters_ref(),
+                              init_non_voter => ra_nvid(),
```

Comment:
could we name this `non_voter_id`?

Reply:
I had the key exactly at that, but one consideration made me change it: unlike UID this one can (hypothetically) be changed with follow-up cluster changes. How about `initial_non_voter_id`?